testing/framework/TestCmdTests.py

```diff
@@ -551,255 +602,309 @@ def test_diff_re_static_method(self) -> None:
     '5c5',
     "< 'f1'",
     '---',
-    "> 'f2'"
+    "> 'f2'",
 ]
 assert result == expect, result
 sys.exit(0)
-""")
-
+""",
+        )
 
 
 class diff_stderr_TestCase(TestCmdTestCase):
     def test_diff_stderr_default(self) -> None:
         """Test diff_stderr() default behavior"""
-        self.popen_python(fr"""import sys
-sys.path = [r'{self.orig_cwd}'] + sys.path
+        self.popen_python(
+            indata=f"""\
+import sys
 import TestCmd
+
+sys.path = [r'{self.orig_cwd}'] + sys.path
+
 test = TestCmd.TestCmd()
-test.diff_stderr('a\nb1\nc\n', 'a\nb2\nc\n')
+test.diff_stderr('a\\nb1\\nc\\n', 'a\\nb2\\nc\\n')
 sys.exit(0)
 """,
-                          stdout="""\
+            stdout="""\
 2c2
 < b1
 ---
 > b2
-""")
+""",
+        )
 
     def test_diff_stderr_not_affecting_diff_stdout(self) -> None:
         """Test diff_stderr() not affecting diff_stdout() behavior"""
-        self.popen_python(fr"""
+        self.popen_python(
+            indata=f"""\
 import sys
-sys.path = [r'{self.orig_cwd}'] + sys.path
 import TestCmd
+
+sys.path = [r'{self.orig_cwd}'] + sys.path
 test = TestCmd.TestCmd(diff_stderr='diff_re')
 print("diff_stderr:")
-test.diff_stderr('a\nb.\nc\n', 'a\nbb\nc\n')
+test.diff_stderr('a\\nb.\\nc\\n', 'a\\nbb\\nc\\n')
```

Comment:
Is this right? it's escaping the newline so it'll show up as \n instead of being a newline?

Reply:
missed that one... I didn't think reformatting would change that.  Will check into.

Reply:
Hmm, think the change is actually correct, as it's a string-in-a-string problem. Most of the other code snippets run via `self.popen_python` already did this escape-escaping.

Reply:
There's more than a handful of these changes in this file if I remember correctly. Were they broken before and just never tested?